app/services/ga_tracker.rb

```diff
@@ -18,24 +18,25 @@ def set_visit_processing_time_cookie
 
   def send_unexpected_rejection_event
     if visit_rejected_unexpectedly?
-      send_data(build_event_payload('Manual rejection',
+      send_data(build_event_payload(ga_cookie, 'Manual rejection',
         visit.rejection.reasons.sort.join('-')))
     end
   end
 
   def send_rejection_event
     if visit.rejected?
-      send_data(build_event_payload('Rejection',
+      send_data(build_event_payload(ga_cookie, 'Rejection',
         visit.rejection.reasons.sort.join('-')))
     end
   end
 
   def send_request_event
-    send_data(build_event_payload('Request', visit.slots.count))
+    send_data(build_event_payload('GA1.1.123456789.0123456789',
```

Comment:
@chrishanes @KPobeeNorris - shouldn't the first parameter refer to 'ga-cookie'?

Reply:
No. The cookie will not exist, so we need to set a static one